package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -299,7 +299,7 @@ def phase_cp(universe, seg, i):
          + (r3_d * cos(4 * pi * 2.0 / 5.0)) + (r4_d * cos(4 * pi * 3.0 / 5.0))
          + (r5_d * cos(4 * pi * 4.0 / 5.0))) * sqrt(2.0 / 5.0)
 
-    phase_ang = (atan2(D, C) + (pi / 2.)) * 180. / pi
```

Comment:
Remove the imported `atan2` from the import line at the top
```python
from math import pi, sin, cos, atan2, sqrt, pow
```

Reply:
> Remove the imported `atan2` from the import line at the top
> 
> ```python
> from math import pi, sin, cos, atan2, sqrt, pow
> ```


Actually on line 371 atan2 is being used. But that is not generating any warnings so I didn't change that.
```python
phase_ang = atan2(B, A) * 180. / pi
```

Reply:
Can you replace that one, too, please, for consistency?

Reply:
> Can you replace that one, too, please, for consistency?

Yes no problem. I have made the necessary changes